Bust Config.rb

```diff
@@ -166,6 +167,13 @@ def custom_bust_display_options
         nil,
         face_name = 'Vera emo'
       ]
+    elsif character_name == 'Uyae' && $game_switches[1481] # YHILIN III
+      # fix up Uyae's clothes after returning from first Zirantia trip
+      return [
+        nil,
+        nil,
+        face_name = 'Uyae2 emo'
+      ]
```

Comment:
This block is in the same place as the one for Simon's topless form - I'll fix the merge conflict in whichever branch gets merged last.